client/layout/head/index.jsx

```diff
@@ -35,7 +35,12 @@ Head.propTypes = {
 	title: React.PropTypes.string,
 	description: React.PropTypes.string,
 	canonicalUrl: React.PropTypes.string,
```

Comment:
Maybe we should go with `url` instead of `canonicalUrl` just to be consistent with Open Graph naming?
I think that this is also why we had `site_name` - but that is just a detail.